src/osx/carbon/dcscreen.cpp

```diff
@@ -26,12 +26,14 @@ wxIMPLEMENT_ABSTRACT_CLASS(wxScreenDCImpl, wxWindowDCImpl);
 wxScreenDCImpl::wxScreenDCImpl( wxDC *owner ) :
    wxWindowDCImpl( owner )
 {
+#ifndef __WXOSX_IPHONE__
```

Comment:
```suggestion
#if !wxOSX_USE_IPHONE
```
for consistency with the other checks in this file.

Reply:
agreed , fine. i will change this in the upcoming commit